deployment/quick-start/services-configuration.yaml.template

```diff
@@ -15,131 +15,110 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-cluster:
 
-  clusterid: pai-example
 
-  # HDFS, zookeeper data path on your cluster machine.
-  data-path: "/datastorage"
+cluster:
+  common:
+    clusterid: pai-example
+
+    # HDFS, zookeeper data path on your cluster machine.
+    data-path: "/datastorage"
 
   # the docker registry to store docker images that contain system services like frameworklauncher, hadoop, etc.
-  docker-registry-info:
+  docker-registry:
 
     # If public, please fill it the same as your username
-    docker-namespace: openpai
+    namespace: openpai
 
     # E.g., gcr.io. If public, fill docker_registry_domain with word "public"
```

Comment:
maybe this comment and following comments are outdated?

Reply:
Updated